docker-compose.yml

```diff
@@ -1,15 +1,16 @@
-# docker-compose.yml
 version: '3'
 networks:
   todoapp-net:
 services:
   pythonapp:
-    image: todoapp:2.0.0
+    image: todoapp:latest
     build:
       context: .
       dockerfile: Dockerfile
     ports:
       - "8080:8080"
+    env_file:
+      - .env 
```

Comment:
It's a bit unclear why the .env file is missing and it's added to .gitignore, but there is a .env.example.

Reply:
Ok. got your point